### DISCUSSION TITLE
[1408230] Walkabout Mini Golf [仅本体]

### DISCUSSION THREAD

**Discussion by Liangyu-G** [pjy612/SteamManifestCache#350] [Share]
appid:
`1408230`
keys:
```
1408231;801d8b1cbfc15f01db9306fd2300d43d12ac7cfa46ffde9b86425538d1966813
```
code:
`844AC319FF5E335E`
email:
2995519244@qq.com
文件:
[[1408230] Walkabout Mini Golf [仅本体].zip](https://github.com/pjy612/SteamManifestCache/files/13878157/1408230.Walkabout.Mini.Golf.zip)

**Comment by pjy612** [comment]
抽空 编辑下帖子 把 密钥在帖子贴一下

**Reply by Liangyu-G** [reply]
已更新
